week3/api-services/property_search_api.py

```
@@ -0,0 +1,399 @@
+                "max_lat": max_lat,
+                "min_lon": min_lon,
+                "max_lon": max_lon
+            },
+            "filters": {
+                "max_crime": max_crime,
+                "property_type": property_type
+            },
+            "query_time_ms": round(query_time, 3)
+        }
+        
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)